codespell_lib/data/dictionary.txt

```diff
@@ -20740,6 +20740,7 @@ namemespace->namespace
 namepace->namespace
 namepsace->namespace
 namepsaces->namespaces
+namesapce->namespace
```

Comment:
@penguin42 If you do not mind adding these as well.
```txt
namesapced->namespaced
namesapces->namespaces
```
* https://github.com/search?q=namesapce&type=code
* https://github.com/search?q=namesapced&type=code
* https://github.com/search?q=namesapces&type=code

Reply:
Done.